keyboards/handwired/dactyl_manuform/5x6/keymaps/333fred/keymap.c

```diff
@@ -0,0 +1,80 @@
+     _______, _______, _______, _______, _______, _______,                       _______, _______, _______, _______, _______, _______,
+     _______, _______, _______, _______, _______, _______,                       _______, _______, _______, _______, _______, _______,
+     _______, _______, _______, _______, _______, _______,                       _______, _______, _______, _______, _______, _______,
+     _______, _______, _______, _______, _______, _______,                       _______, _______, _______, _______, _______, _______,
+                       KC_F7,   KC_F8,                                                             _______, _______,
+                                         _______, _______,                       _______, _______,
+                                                  _______, _______,     _______, _______,
+                                                  _______, _______,     _______, _______
+  ),
+
+  [SYMB] = LAYOUT_5x6(
+     _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                         KC_F6,   KC_F7,   KC_F8, KC_F9,  KC_F10,  KC_F11,
+     _______, KC_EXLM, KC_AT,   KC_LPRN, KC_RPRN, KC_PIPE,                       _______, KC_7,    KC_8,  KC_9,   _______, KC_F12,
+     _______, KC_HASH, KC_DLR,  KC_LCBR, KC_RCBR, KC_GRV,                        _______, KC_4,    KC_5,  KC_6,   _______, _______,
+     _______, KC_PERC, KC_CIRC, KC_LBRC, KC_RBRC, KC_TILD,                       _______, KC_1,    KC_2,  KC_3,   _______, _______,
+                       KC_F7,   KC_F8,                                                             KC_0,  KC_ENT,
+                                         _______, _______,                       KC_MPLY, KC_MNXT,
+                                                  _______, _______,     KC_MPRV, _______,
+                                                  _______, _______,     KC_VOLU, KC_VOLD
+  ),
+
+  [VIM] = LAYOUT_5x6(
+     _______, RESET,    _______,   _______, _______, _______,                       _______, _______, _______, _______, RESET,   _______,
+     _______, _______,  _______,   _______, KC_LSFT, _______,                       _______, _______, _______, _______, _______, _______,
+     _______, M(DLEFT), M(DRIGHT), KC_LCTL, KC_LGUI, _______,                       KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, _______, _______,
+     _______, _______,  _______,   _______, _______, _______,                       _______, _______, _______, _______, _______, _______,
+                        _______,   _______,                                                            _______, _______,
+                                           _______, _______,                       _______, _______,
+                                                    _______, _______,     _______, _______,
+                                                    _______, _______,     _______, _______
+  ),
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
Didn't want to convert to `process_record_user`?

Reply:
It works. Until we actually remove it, I'd prefer to just keep it as is.